README.md

```diff
@@ -369,17 +482,17 @@ In order of priority:
   * Add `SOUND_VOLUME` to set the sound volume by label
   * Add `SOUND_STOP` to stop playing sound by label and delete the sound label

```

Comment:
Since `SOUND_STOP` is implemented now I suggest that you document it and remove it here